packages/compass-settings/src/components/settings/feature-preview.tsx

```diff
@@ -47,12 +39,20 @@ export function useShouldShowFeaturePreviewSettings(): boolean {
   // - or if:
   //   - we are in a development environment or 'showDevFeatureFlags' is explicitly enabled
   //   - and there are feature flags in 'development' stage.
+  //   - AI feature flag is enabled
```

Comment:
This only triggers when a user has enabled the AI feature flag through the dev feature flag setting, right?

Should this just check whether any preview feature flag is enabled at all? This code should kind of be agnostic about what the actual flags are

Reply:
Hmm, I'm not sure I follow. As far as I understand this particular feature flag will never leave dev stage and for public beta we will just always show the "Feature Flag" section as this is where atlas login ended up in for now while it's only used for AI feature

Reply:
So the `enableAIExperience` is just going to go away at that point? Then it’s fine either way :+1:

Reply:
I think so, yeah, I'll double-check! My assumption was that already for public beta we will just show this always. And the opt-in toggle is not really a preference / ff as it's supposed to be mapped to the atlas user id and so doesn't really fit the preferences store (although I'm open to thoughts / suggestions if you have opinions on that!)